demojsrouting/README.md

```diff
@@ -0,0 +1,26 @@
+# Demo js routing
+
+## About
+
+Explains how to use javascript routing component in a module
+
+### Supported PrestaShop versions
+
+Compatible with and 1.7.7.0 and above versions.
```

Comment:
ok so Idk what version its actually compatable with. Definately not 177, because the Router component was not added to defined components list.. Its only pending in develop now, so we will see when its merged :sweat: (Its possible to push it through 178 as a bug fix, but if not. then it will only come with a new major I guess?)

Reply:
Let's wait patiently for the answer :)

Reply:
I think we should push it to 1.7.8, this looks like a key feature to me that we forgot

Reply:
@NeOMakinG Did you meet success? 😊

Reply:
I think yes : https://github.com/PrestaShop/PrestaShop/pull/24682